src/lib/constants.js

```diff
@@ -33,4 +36,33 @@ module.exports = Object.freeze({
   ],
   ENUM_POST_TYPE: ['Request', 'Donate'],
   ENUM_POST_STATUS: ['Published', 'Verified'],
+
```

Comment:
extra new line

Reply:
@khaldarov I meant that it was an extra line, so please remove the comment and remove the extra line too

Reply:
My bad. Now fixed.